node/src/gql/mutation/me/update-question.ts

```diff
@@ -50,41 +49,10 @@ export const updateQuestion = {
     args: { question: QuestionUpdateInput },
     context: { user: User }
   ): Promise<Question> => {
-    const questionUpdate: QuestionUpdateInput = args.question;
-    for (const [i, topic] of (questionUpdate.topics || []).entries()) {
-      topic._id = idOrNew(topic._id);
-      const t = await TopicModel.findOneAndUpdate(
-        {
-          _id: topic._id,
-        },
-        {
-          $set: {
-            ...topic,
-          },
-        },
-        {
-          new: true,
-          upsert: true,
-        }
-      );
-      questionUpdate.topics[i] = t;
-    }
-
-    const question: any = { ...questionUpdate };
-    if (questionUpdate.topics) {
-      question._id = idOrNew(questionUpdate._id);
-      question.topics = questionUpdate.topics.map((t) => t._id);
-    }
-
+    const { _id, props } = toUpdateProps<Question>(args.question);
     return await QuestionModel.findOneAndUpdate(
-      {
-        _id: question._id,
-      },
-      {
-        $set: {
-          ...question,
-        },
-      },
+      { _id: _id },
```

Comment:
can this not be just:

```typescript
{ _id }, { $set: props }
```

...is not a big deal, but don't want to get in the habit of doing that copy if we don't need it. Or if we do need it, I should learn that